### PR TITLE
[docs] - Dagster Cloud full deployment docs

### PR DESCRIPTION
**Pull request by erinkcochran87**
### Summary & Motivation

This PR adds content about Dagster Cloud full deployments to the docs, specifically:

- [Managing deployments](https://dagster-git-erin-managing-deployments-elementl.vercel.app/dagster-cloud/developing-testing/managing-deployments) (**Sidenav**: **Deployment > Cloud > Developing & testing > Full deployments > Management**)
- [Deployment settings reference](https://dagster-git-erin-managing-deployments-elementl.vercel.app/dagster-cloud/developing-testing/deployment-settings-reference) - I moved this content into its own file, as the original guide was getting pretty cluttered.
   (**Sidenav**: **Deployment > Cloud > Developing & testing > Full deployments > Settings**)

### How I Tested These Changes

`yarn test`

### CODE REVIEW

File: docs/content/dagster-cloud/developing-testing/managing-deployments.mdx [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+## Configuring deployment settings
+
+<Note>
+  <a href="/dagster-cloud/account/managing-users#understanding-user-permissions">
+    Organization Admin permissions
+  </a>{" "}
+  are required to modify deployment settings.
+</Note>
+
+Deployment settings can be configured in the Dagster Cloud interface or using the dagster-cloud CLI. Refer to the [deployment settings reference](/dagster-cloud/developing-testing/deployment-settings-reference) for more info about individual settings.
```
Comment (gibsondan) [inline docs/content/dagster-cloud/developing-testing/managing-deployments.mdx:107]: should we make it clear that these settings only apply to full deployments? (even though this is in the Full Deployments section)

File: docs/content/dagster-cloud/developing-testing/managing-deployments.mdx [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+     </ul>
+    </li>
+    <li>Click the <strong>Settings</strong> button next to the deployment you want to configure.</li>
+    <li>In the window that displays, configure settings for the deployment.</li>
+    <li>When finished, click <strong>Save deployment settings</strong>.</li>
+   </ol>
+  </TabItem>
+  <TabItem name="dagster-cloud CLI">
+<Note>
+<strong>Note:</strong> <code>dagster-cloud</code> 0.13.14 or later must be installed to run the CLI. Agent and/or job code doesn't need to be upgraded.
```
Comment (gibsondan) [inline docs/content/dagster-cloud/developing-testing/managing-deployments.mdx:128]: this version is so old at this point that I wonder if we even need to say this

File: docs/content/dagster-cloud/developing-testing/managing-deployments.mdx [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+    Organization Admin permissions
+  </a>{" "}
+  are required to modify deployment settings.
+</Note>
+
+Deployment settings can be configured in the Dagster Cloud interface or using the dagster-cloud CLI. Refer to the [deployment settings reference](/dagster-cloud/developing-testing/deployment-settings-reference) for more info about individual settings.
+
+<TabGroup>
+  <TabItem name="Dagster Cloud">
+   To configure deployment settings in the Dagster Cloud UI:
```
Comment (gibsondan) [inline docs/content/dagster-cloud/developing-testing/managing-deployments.mdx:111]: there's also a gear in the deployment switcher that lets you access this

**Review by gibsondan — APPROVED** [review]
apologies for the delay here - this looks great, feedback is all fairly rote (the most substantive thing is probably the part about branch deployments not spinning down on merge)

File: docs/content/dagster-cloud/developing-testing/managing-deployments.mdx (hunk 2 above)
Comment (erinkcochran87) [reply docs/content/dagster-cloud/developing-testing/managing-deployments.mdx:128]: I'm honestly not sure. Is there a way we can tell if anyone is still using it?

File: docs/content/dagster-cloud/developing-testing/managing-deployments.mdx (hunk 1 above)
Comment (erinkcochran87) [reply docs/content/dagster-cloud/developing-testing/managing-deployments.mdx:107]: I'm inclined to say it's not necessary, given that we specifically state this guide is for full deployments before this.

**Pull request merged**